trio/_tests/test_threads.py

```diff
@@ -426,6 +461,11 @@ async def test_run_in_worker_thread_limiter(MAX, cancel, use_default_limiter):
         # Mutating them in-place is OK though (as long as you use proper
         # locking etc.).
         class state:
+            if TYPE_CHECKING:
```

Comment:
No need to guard this, they'll just set `__annotations__`?

Reply:
That should be correct yes, as far as I am aware.

Reply:
correct, thanks~